Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "oxnet"
 version = "0.1.1"
```

Comment:
do we want to bump this to 0.1.2?

Reply:
yeah, I should bump this up for release (haven't done this for small libs for us yet).

Reply:
oxnet is one of the few small libs we publish (i.e. to my knowledge)